code/src/components/Checkbox.js

```diff
@@ -0,0 +1,31 @@
+import React from'react';
+
+import 'components/CheckboxCss.css'
+
+// I cannot explain this function component.
+export const Checkbox = ({userLikes, onLikesChange}) => {
+
+  return (
+    <div>
+      <h2 tabindex="0">4. What else do you like?</h2>
+      <div className="checkbox-container">
+        <label htmlFor="likeKittens">Kittens</label>
+        <input
+          id="likeKittens"
+          type="checkbox"
+          checked={userLikes.includes('kittens')}
+          onChange={() => onLikesChange('kittens')} 
+        />
+        <span className="checkbox-label"></span>
+        <label htmlFor="likeStrawberries">Strawberries</label>
+        <input
+          id="likeStrawberries"
+          type="checkbox"
+          checked={userLikes.includes('strawberries')}
+          onChange={() => onLikesChange('strawberries')}
+        />
```

Comment:
When I was using the keyboard to tab I found that I couldn't select either of the checkboxes using the arrows. I tried to do this by clicking enter when I had tabbed on to wither of the buttons, but this  submitted the form. Did you create custom checkboxes? Not sure what the solution would be, but I know that I also had issues selecting my checkboxes using the keyboard, as I had used customised checkboxes. Something the teacher can confirm what to do in this situation?

Reply:
I think you should use the tab key to bring the cursor to the check-box and then hit the space bar on the keyboard to check or uncheck it. And Claire, I think the same goes for your radio buttons as we discussed - if you use tab and space they are selectable!